reference_data/management/tests/run_reference_data_database_backup_2_3_tests.py

```diff
@@ -1,24 +1,21 @@
 import mock
-from io import BytesIO
 import datetime
 
 from django.core.management import call_command
 from django.test import TestCase
 
 
+@mock.patch('reference_data.management.commands.run_reference_data_database_backup.logger')
 @mock.patch('reference_data.management.commands.run_reference_data_database_backup.os')
 @mock.patch('reference_data.management.commands.run_reference_data_database_backup.datetime')
 class RunReferenceDataDbBackupTest(TestCase):
     fixtures = ['users', 'reference_data']
 
-    def test_reference_data_db_backup_command(self, mock_datetime, mock_os):
-        out = BytesIO()
-
-        mock_datetime.datetime.now.return_value = datetime.datetime(2020, 4, 27, 20, 16, 01)
+    def test_reference_data_db_backup_command(self, mock_datetime, mock_os, mock_logger):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5795824)

Reply:
not relevant